templates/tools/dockerfile/test/python_testing_x64/Dockerfile.template

```diff
@@ -0,0 +1,34 @@
+%YAML 1.2
```

Comment:
Filename: Let's not use "testing" but "stretch", what's marked as debian testing changes over time, "stretch" is always the same distro. Other docker images are also using "jessie", not "stable". 

A bit unrelated: "stretch" is not testing anymore, it's the new stable. let's just use "stretch".